package.json

```diff
@@ -39,7 +39,8 @@
     "check:types": "yarn workspaces foreach run check:types",
     "lint": "eslint .",
     "dev:container": "yarn workspace container run dev",
-    "dev:demo": "yarn workspace demo run dev"
+    "dev:demo": "yarn workspace demo run dev",
+    "deploy": "./scripts/deploy.mjs"
```

Comment:
maybe the naming is a bit confusing, what about calling it versioning or bump-version

Reply:
Hmmm. You're right. This doesn't deploy, but only bumps the version. And the deployment actually happens after creating gh release. Then...  `bump-version` sounds good to m.e